cdc/processor/pipeline/actor_node_context.go

```diff
@@ -20,30 +20,31 @@ import (
 	"github.com/pingcap/log"
 	"github.com/pingcap/tiflow/pkg/actor"
 	"github.com/pingcap/tiflow/pkg/actor/message"
+	"github.com/pingcap/tiflow/pkg/config"
 	"github.com/pingcap/tiflow/pkg/context"
 	"github.com/pingcap/tiflow/pkg/pipeline"
 	"go.uber.org/zap"
 )
 
-// send a tick message to actor if we get 32 pipeline messages
-const messagesPerTick = 32
+// send a tick message to actor if we get 32 pipeline events
```

Comment:
It is recommended to start a go comment with a variable name or a function name.

Reply:
done